common/src/main/resources/assets/fireplacelib/lang/es_es

```diff
@@ -0,0 +1,15 @@
+{
+  "modmenu.descriptionTranslation.fireplacelib": "${mod_description}",
```

Comment:
This substitutes in the description in English from gradle.properties. you might want to put the translated version of that here instead